.travis.yml

```diff
@@ -1,11 +1,13 @@
+sudo: false
 language: node_js
 git:
   depth: 10
 node_js:
   - "0.12"
 before_install:
-  - travis_retry sudo pip install -r test-infra/requirements.txt
-  - rvm use 1.9.3 --fuzzy
+  - export PATH=$HOME/.local/bin:$PATH
```

Comment:
We only use the libraries that pip installs, not the executables, so this should be unnecessary